### PR TITLE
Data 2106 google o auth

### PR DESCRIPTION
**Pull request by J314Y**
@yixu34 @ravindrv  check staging

### CODE REVIEW

**Comment by J314Y** [comment]
removed the annotations, staging works

File: redash/query_runner/presto.py [hunk 1]
```diff
@@ -97,10 +97,15 @@ def get_schema(self, get_stats=False):
         return schema.values()
 
     def run_query(self, query, user):
+        if user is None:
```
Comment (yixu34) [inline redash/query_runner/presto.py]: I'd do something more pythonic here, maybe `username = user.name if user else "GeneralQueryRunner"`

**Comment by yixu34** [comment]
Well that was simple enough I guess. LGTM, minor comment though.

**Comment by ravindrv** [comment]
🆗 🚢

**Pull request merged**